frontend/src/components/table/classes.ts

```diff
@@ -68,7 +68,7 @@ export const tableClass = (key: string, value?: string) => {
     case DashboardColumns.REMOTE:
       return tableClasses.remote;
     case DashboardColumns.AVAILABILITY:
-      return getAvailabilityClass(value);
+      return getAvailabilityClass(value?.split(' ')[0]);
```

Comment:
I don't like this, but I don't see an easy way around

Reply:
I know. It's not great.